composer.json

```diff
@@ -22,8 +22,8 @@
     "require-dev": {
         "phpro/grumphp": "^0.17.1 || ^0.19.0",
         "phpstan/phpstan": "^0.11.15 || ^0.12.0",
-        "phpunit/phpunit": ">=7.5",
-        "squizlabs/php_codesniffer": "^3.0",
+        "phpunit/phpunit": ">=9.0",
```

Comment:
Since the package requires `php-7.4` version at least, it's time to upgrade PHPUnit version to `9.0` version at least.

Reply:
Agree. but better we can get version values from [antidot-fw/framework](https://github.com/antidot-framework/antidot-framework/blob/fd2a41645bc1ec1f6d1022be59134efd82a55112/composer.json#L31) package's composer.json

Reply:
That's right. It should be same as the `antidot-fw/framework` package because of consistency :).